yaral/2_10_org_admin_permissions_granted.yaral

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+rule workspace_user_granted_admin_privilege {
+  meta:
+    author = "Google Cloud Security"
+    description = "A user is granted an admin privilege"
+    severity = "HIGH"
+    url = "https://developers.google.com/admin-sdk/reports/v1/appendix/activity/admin-delegated-admin-settings"
+
+  events:
+    $workspace.metadata.vendor_name        = "Google Workspace"
+    $workspace.metadata.product_name       = /admin/ nocase
+    $workspace.metadata.product_event_type = /assign_role/ nocase
```

Comment:
This looks equivalent to above `yaral/2_01_super_admin_or_admin_permissions_granted.yaral` rule.
2.10 use case should trigger on setIAMPolicy event, not assignRole which is slightly different. (see equivalent [2.10 SQL](https://github.com/GoogleCloudPlatform/threat-detection-as-code/blob/main/sql/2_10_org_admin_permissions_granted.sql#L24))

Reply:
Thanks, I have corrected this